app/src/androidTest/java/com/sdp/movemeet/viewTest/chatTest/MessageHolderTest.java

```diff
@@ -0,0 +1,5 @@
+package com.sdp.movemeet.viewTest.chatTest;
+
+public class MessageHolderTest {
```

Comment:
This test is still empty ?

Reply:
Yes, for the moment, I have still no idea how to test this class ^^' maybe you have an idea?

Reply:
No yet sorry